tools/scylla-sstable.cc

```diff
@@ -3013,7 +3013,9 @@ Validate the specified sstables:
             if (!schema_sources) {
                 sst_log.debug("No user-provided schema source, attempting to auto-detect it");
                 schema_with_source = try_load_schema_autodetect(app_config, dbcfg);
-            } else if (schema_sources == 1) {
+            } else if (schema_sources == 1 || (schema_sources == 2 && app_config.contains("scylla-yaml-file"))) {
+                // We make an exception for the case where 2 schema sources are provided, but one of them is scylla-yaml file.
+                // We want to always accept the --scylla-yaml-file option.
```

Comment:
could you help me understand what "e.v." stands for in the commit message?

Reply:
It is supposed to be "e.g." Will fix.